src/st/no3/377D4.c

```diff
@@ -181,7 +198,24 @@ void func_801BBDC0(Entity* arg0) {
     }
 }
 
-INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801BBE70);
+void func_801BBE70(Entity* arg0) {
+    u16 temp_v0;
+
+    if (arg0->step == 0) {
+        InitializeEntity(&D_80180B18);
```

Comment:
Any parameter to `InitializeEntity` is an array of values. So `InitializeEntity(&D_80180B18)` becomes `InitializeEntity(D_80180B18)`(`&` removed) and `extern u16 D_80180B18;` becomes `extern u16 D_80180B18[];`